ppr-ui/src/views/dashboard.vue

```diff
@@ -1,42 +1,82 @@
 <template>
   <v-container fluid class="pa-10">
-    <v-row no-gutters>
-      <v-col cols="12" class="search-title">
-        <b>Search</b>
-      </v-col>
-    </v-row>
-    <v-row no-gutters>
-      <v-col cols="12" class="search-info pt-2">
-        <span>
-          Select a search category and then enter a value to search.
-          <b>Note:</b>
-          Each search incurs a fee (including searches that return no results).
-        </span>
+    <v-row>
+      <v-col>
+        <v-row no-gutters>
+          <v-col cols="12" class="search-title">
```

Comment:
Right on! 
The biggest reason name request wasn't responsive was due to the col definitions, or lack of.